pkg/api/dataproducts.go

```diff
@@ -0,0 +1,286 @@
+package api
```

Comment:
In dataproducts case we cant, because the persistent data model is quite different that we cannot simply reuse at least with go (if we use java/kotlin with hibernate, the thing may be different). The query for dataproducts returns a couple of rows where several of them constitute a  dataset object, which have to be integrated programmatically